cmd/ocm/edit/ingress/cmd.go

```diff
@@ -55,6 +67,12 @@ const (
 	namespaceOwnershipPolicyFlag  = "namespace-ownership-policy"
 	clusterRoutesHostnameFlag     = "cluster-routes-hostname"
 	clusterRoutesTlsSecretRefFlag = "cluster-routes-tls-secret-ref"
+	componentRoutesFlag           = "component-routes"
+
+	amountOfParametersPerComponentRoute = 2
+	hostnameParameter                   = "hostname"
+	//nolint:gosec
+	tlsSecretRefParameter = "tlsSecretRef"
```

Comment:
nit: indenting seems off

Reply:
Oh it's because of the nolint comment, won't be able to resolve it